io/net/bonding.py

```diff
@@ -520,6 +528,19 @@ def test_cleanup(self):
             except Exception:
                 self.log.info("backup file not availbale, could not restore file.")
 
+        detected_distro = distro.detect()
+        if detected_distro.name == "rhel":
+            cmd = "systemctl restart NetworkManager.service"
+        elif detected_distro.name == "Ubuntu":
+            cmd = "systemctl restart networking"
+        else:
+            cmd = "systemctl restart network"
+        if process.system(cmd, shell=True, ignore_status=True) != 0:
+            self.fail("Failed to restart the network service on host")
+        output = self.session_public_ip.cmd(cmd)
+        if not output.exit_status == 0:
```

Comment:
Network restart is performed after executing all the commands on peer lpar,  hence it doesn't affect the other sessions. And the test terminates after network reset. For new test session is created again .

As per the OS is different. Session restart is needed mainly for host lpar. just to avoid any network issue on peer network reset is performed on peer as @vaishnavibhat has suggested. 
Please let me know if we have cases of different OS and is it really need to start the network on peer lpar?

Reply:
@abdhaleegit  Not doing network reset on peer. Network reset is performed only on host lpar.